Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -1272,17 +1272,14 @@ instance {R} [Semiring R] : NonUnitalSubsemiringClass (Ideal R) R where
   mul_mem _ hb := Ideal.mul_mem_left _ _ hb
 instance {R} [Ring R] : NonUnitalSubringClass (Ideal R) R where
 
-lemma Ideal.exists_subset_radical_span_sup_span_of_subset_radical_sup {R : Type*} [CommSemiring R]
+lemma Ideal.exists_subset_radical_span_sup_of_subset_radical_sup {R : Type*} [CommSemiring R]
     (s : Set R) (I J : Ideal R) (hs : s ⊆ (I ⊔ J).radical) :
-    ∃ (t₁ t₂ : s → R), Set.range t₁ ⊆ I ∧ Set.range t₂ ⊆ J ∧
-      s ⊆ (span (Set.range t₁) ⊔ span (Set.range t₂)).radical := by
+    ∃ (t : s → R), Set.range t ⊆ I ∧ s ⊆ (span (Set.range t) ⊔ J).radical := by
```

Comment:
Why is this changed? I don't think this is more general?

Reply:
It's a technical lemma extracted from the proof of `height_le_spanRank_toENat_of_mem_minimal_primes` by @chrisflav and [modified](https://github.com/leanprover-community/mathlib4/pull/23778/commits/d945aae72d33634db6f03545ba408931bb2b543d) by @xyzw12345 in #23778. We don't actually need to change J to `span (range _)` to make the proof work, and keeping J makes the proof simpler.